src/main/java/io/split/android/client/storage/db/MySegmentDao.java

```diff
@@ -0,0 +1,22 @@
+package io.split.android.client.storage.db;
+
+import androidx.lifecycle.LiveData;
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+import androidx.room.Update;
+
+import java.util.List;
+
+import io.split.android.client.dtos.MySegment;
+
+@Dao
+public interface MySegmentDao {
```

Comment:
Will be possible to add `remove` operation?

Reply:
@mmelograno , thanks for the review!. For now we don't have a specific logic intended to use a remove method here.
A good use case for that would be a user key deprecation policy but currently we don't have a spec for that. Maybe we can add it. What do you think @mredolatti ?